asmcomp/i386/proc.ml

```diff
@@ -107,7 +107,7 @@ let word_addressed = false
    These globals are denoted by Incoming and Outgoing stack locations
    with negative offsets, starting at -64.
    Unlike arguments passed on stack, arguments passed in globals
-   do not prevent tail-call elimination.  The caller stores arguments
+   do not prevent tailcall elimination.  The caller stores arguments
```

Comment:
Fight (against entropy), battle, dragons : waouh, 
are we in a 'Game of Thrones' episode (season 7 ?)

Well, I revert 'marshaling/unmarshaling' to 'marshalling/unmarshalling' (mshinwell's preference ... )
and, following your suggestion I revert/correct some 'tailcall's (nouns/adjectives).

**Note:** petit laïus "à la gasche" et en V.O. sur 

> 'the battle you want to fight'

Après avoir pratiquement lu tous les commentaires des sources OCaml (je suis peut-être le seul au monde à l'avoir fait, mais c'est bon, on en survit - d'ailleurs, au départ, j'espérais trouver quelques remarques rigolotes, mais non, pratiquement tout est désespérement technique. grrr ...) je peux assurer que la 'guerre' n'est pas là où on pourrait le croire.

D'abord les typos 'dragons'. Un chiffre : 40.
La question les concernant peut ,en gros, se résumer à : pour quelques termes techniques
très récents , doit-on utiliser l'Anglais (avec ses 's' et autres 'll') ,l'Américain (avec 'z' et 'l')
ou bien un joyeux mélange ? Cette dernière solution est la meilleure ? OK.
Puisque tout est accepté/acceptable tous les problèmes disparaissent et il n'y a aucune guerre, aucun dragon.

Par contre les 'vrais' typos ... déjà leur nombre : près de 400 . Ouch ! .
Ensuite, ils sont protéïformes, ils peuvent avoir une longue durée de vie
(certains doivent avoir 15/20 ans, non ?), ils ne sont pas toujours facile à voir
(des relectures du code vont obligatoirement en faire apparaître d'autres),
et donc, si on parle 'guerre' c'est véritablement contre eux qu'elle a lieu.

_Rem:_ j'ai vu qu'il existe un script 'check-typo' (dans le répertoire tools).
Serait-il possible d'ajouter des règles permettant la correction automatique de
quelques typos fréquents (occurence ...) ? 
Ou faut-il continuer à chercher "à la main" ?